src/Apps/Artist/Components/__tests__/ArtistConsignButton.test.tsx

```diff
@@ -0,0 +1,115 @@
+        )
+      },
+      query: graphql`
+        query ArtistConsignButton_Test_Query($artistID: String!)
+          @raw_response_type {
+          artist(id: $artistID) {
+            ...ArtistConsignButton_artist
+          }
+        }
+      `,
+      variables: {
+        artistID: "alex-katz",
+      },
+      mockData: response,
+    })
+  }
+
+  describe("Top 20 Button", () => {
+    const response = {
+      artist: {
+        name: "Alex Katz",
+        href: "/artist/alex-katz",
+        image: {
+          cropped: {
+            url:
+              "https://d7hftxdivxxvm.cloudfront.net?resize_to=fill&width=75&height=66&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FbrHdWfNxoereaVk2VOneuw%2Flarge.jpg",
+          },
+        },
+        id: "QXJ0aXN0OjRkOGQxMjBjODc2YzY5N2FlMTAwMDA0Ng==",
+      },
+    }
+
+    it("renders desktop version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "md", response })
+      expect(wrapper.find("Image").length).toEqual(1)
+      expect(wrapper.text()).toContain("Sell your Alex Katz")
+      expect(wrapper.find("RouterLink").html()).toContain(
+        `href="/artist/alex-katz/consign"`
+      )
+    })
+
+    it("renders mobile version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "xs", response })
+      expect(wrapper.find("Image").length).toEqual(1)
+      expect(wrapper.text()).toContain("Sell your Alex Katz")
+      expect(wrapper.find("RouterLink").html()).toContain(
+        `href="/artist/alex-katz/consign"`
+      )
+    })
+  })
+
+  describe("Default Button", () => {
+    const response = {
+      artist: {
+        name: "Andy Warhol",
+        href: "/artist/andy-warhol",
+        image: {
+          cropped: {
+            url:
+              "https://d7hftxdivxxvm.cloudfront.net?resize_to=fill&width=75&height=66&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FbrHdWfNxoereaVk2VOneuw%2Flarge.jpg",
+          },
+        },
+        id: "QXJ0aXN0OjRkOGQxMjBjODc2YzY5N2FlMTAwMDA0Ng==",
+      },
+    }
+
+    it("renders desktop version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "md", response })
+      expect(wrapper.find("Image").length).toEqual(0)
+      expect(wrapper.text()).toContain("Sell art from your collection")
+      expect(wrapper.find("RouterLink").html()).toContain(`href="/consign"`)
+    })
+
+    it("renders mobile version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "xs", response })
+      expect(wrapper.find("Image").length).toEqual(0)
+      expect(wrapper.text()).toContain("Sell art from your collection")
+      expect(wrapper.find("RouterLink").html()).toContain(`href="/consign"`)
+    })
+  })
+
+  // TODO: Functionality doesn't yet exist
+  xdescribe("Target Supply Button", () => {
+    it("renders desktop version", async () => {
+      //
+    })
+
+    it("renders mobile version", async () => {
+      //
+    })
+  })
```

Comment:
```suggestion
```

This seems unnecessary to me - what if we change our minds and don't do this button?

Reply:
That makes sense 👍